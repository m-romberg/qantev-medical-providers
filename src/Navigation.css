.Navigation {
    display: flex;
    justify-content: space-between;
    margin: 15px;
}

.Navigation a {
    text-decoration:none;
    color: darkgrey;
    font-size: 19px;
}

.Navigation a:hover {
    color: red;
}