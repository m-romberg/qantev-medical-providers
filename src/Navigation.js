import { NavLink } from "react-router-dom";
import "./Navigation.css";

/** Navigation:
* Visual element with links to home
*
* props:
* - none
* state:
* - none
*  NavLinks
*       to: Home
*/
function Navigation() {
  return (
    <nav className="Navigation Navigation-loggedIn">
      <NavLink to="/home" className="Navigation" end>
        View all providers
      </NavLink>
    </nav>
  );
}

export default Navigation;
