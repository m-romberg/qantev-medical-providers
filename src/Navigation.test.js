import React from "react";
import { render } from "@testing-library/react";
import Navigation from "./Navigation";
import { MemoryRouter } from "react-router";

it("renders without crashing", function() {
  render(<MemoryRouter><Navigation /></MemoryRouter>);
});

it("matches snapshot", function () {
  const { asFragment } = render(
      <MemoryRouter>
          <Navigation />
      </MemoryRouter>,
  );
  expect(asFragment()).toMatchSnapshot();
});