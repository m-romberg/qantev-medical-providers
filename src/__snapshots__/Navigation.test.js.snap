// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <nav
    class="Navigation"
  >
    <a
      class="Navigation"
      href="/home"
    >
      View all providers
    </a>
  </nav>
</DocumentFragment>
`;
